plugins/kotlin/idea/tests/test/org/jetbrains/kotlin/idea/quickfix/K1QuickFixTestGenerated.java

```diff
@@ -13112,6 +13112,22 @@ public void testNormal() throws Exception {
         public void testReturn() throws Exception {
             runTest("testData/quickfix/replaceWithSafeCall/return.kt");
         }
+        
+           @TestMetadata("Test1.kt")
```

Comment:
Test should be generated automatically with "Generate Kotlin Tests (K1 + K2)" run configuration